### PR TITLE
Enhancement: Provide a changelog

### LINKED ISSUE #92: Provide a changelog
#### Steps required to reproduce the problem

1. I open a pull request anywhere suggesting to update `localheinz/composer-normalize`

#### Expected Result

* maintainers have easier means of approving an update since they can review a changelog

#### Actual Result

* maintainers currently only have the means of taking a look at a diff, which is inconvenient

See, for example, https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4023.

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] provides a changelog

Fixes #92.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased](https://github.com/localheinz/composer-normalize/compare/0.9.0...HEAD) ([`0.9.0...master`](https://github.com/localheinz/composer-normalize/compare/0.9.0...master))
+
+### Added
+
+*
```
Comment (keradus) [inline CHANGELOG.md]: is it autogenerated?
if not, i would suggest to skip empty sections,

even keep a changelog skips empty sections in it's own changelog

Comment (localheinz) [reply CHANGELOG.md]: Not (yet) autogenerated, just left them here so I have the right order.

Comment (keradus) [reply CHANGELOG.md]: keep the order and simply leave filled points.

eg, for bugfix releases, you won't have any features, right ? no point to always have them present

Comment (localheinz) [reply CHANGELOG.md]: Thank you, @keradus!

**Comment by localheinz** [comment]
Thank you, @keradus!

**Comment by SpacePossum** [comment]
@localheinz nice man  :+1:

**Comment by localheinz** [comment]
Thank you, @SpacePossum! 🤓

**Pull request merged**